pajbot/modules/casechecker.py

```diff
@@ -37,55 +37,160 @@ class CaseCheckerModule(BaseModule):
             constraints={"min_value": 100, "max_value": 1000},
         ),
         ModuleSetting(
-            key="timeout_duration",
-            label="Timeout duration",
+            key="lowercase_timeouts", label="Enable lowercase timeouts", type="boolean", required=True, default=False
+        ),
+        ModuleSetting(
+            key="lowercase_timeout_duration",
+            label="Lowercase Timeout duration",
             type="number",
             required=True,
             placeholder="",
             default=3,
             constraints={"min_value": 3, "max_value": 120},
         ),
         ModuleSetting(
-            key="online_chat_only", label="Only enabled in online chat", type="boolean", required=True, default=True
+            key="lowercase_timeout_reason",
+            label="Lowercase Timeout Reason",
+            type="text",
+            required=False,
+            placeholder="",
+            default="Too many lowercase characters",
+            constraints={"max_str_len": 500},
+        ),
+        ModuleSetting(
+            key="min_lowercase_characters",
+            label="Minimum amount of lowercase characters before checking for a percentage",
+            type="number",
+            required=True,
+            placeholder="",
+            default=8,
```

Comment:
This changes the default behaviour of the module - we need to either:
a) Not change the default behaviour of the module, or
b) Clearly define in the Changelog that we have changed the default behaviour of the module.
If module settings are checked, but the module is just enabled and it doesn't behave as expected that's not a fun time.